src/extractors/custom/japan.cnet.com/index.js

```diff
@@ -0,0 +1,35 @@
+export const JapanCnetComExtractor = {
+  domain: 'japan.cnet.com',
+
+  title: {
+    selectors: ['.leaf-headline-ttl'],
+  },
+
+  author: {
+    selectors: ['.writer'],
+  },
+
+  date_published: {
+    selectors: ['.date'],
+    format: 'YYYY年MM月DD日 HH時mm分',
+    timezone: 'Asia/Tokyo',
+  },
+
+  dek: null,
+
+  lead_image_url: {
+    selectors: [['meta[name="og:image"]', 'value']],
+  },
+
+  content: {
+    selectors: ['div.article_body'],
+
+    transforms: {
+      '.img-xxxl img': $node => {
```

Comment:
This feels more like a presentational change and something that should be done on the client/consumer side, let's remove this if there is no definitive need for it.

Reply:
Thank you.
Certainly, this was not what the parser should do.